bundles/org.openhab.binding.ondilo/README.md

```diff
@@ -0,0 +1,122 @@
+# Ondilo Binding
+
+This binding integrates Ondilo ICO pool monitoring devices with openHAB, allowing you to monitor and automate your pool environment using openHAB’s rules and UI.
+
+## Supported Things
+
+`bridge:` Represents your Ondilo account (OAuth2)
+
+`ondilo:` Represents an individual Ondilo ICO device
+
+Ondilo ICO Pool as well as Spa devices are supported.
+Chlor as well as salt water.
+
+## Discovery
+
+Ondilo ICOs are discovered automatically after the bridge is authorized and online.
+Each Ondilo ICO will appear as a new Thing in the inbox.
+
+## Thing Configuration
+
+### `bridge` Thing Configuration
+
+- **url**: The URL of the openHAB instance. Required for the redirect during OAuth2 flow (e.g. `http://localhost:8080`)
+- **refreshInterval**: Polling interval of the bridge (i.e. config changes) (default: `1800 s`)
+
+### `ondilo` Thing Configuration
+
+- **id**: The Id of an Ondilo ICO device. Set via discovery service (e.g. `12345`)
+- **refreshInterval**: Polling interval of the device (i.e. measures) (default: `600 s`)
+
+Ondilo ICO takes measures every hour.
+Higher polling will not increase the update interval.
+The requests to the Ondilo Customer API are limited to the following per user quotas:
+
+- 5 requests per second
+- 30 requests per hour
+
+`bridge` Thing performs 1 request per cycle - 2 per hour with default interval.
+
+`ondilo` Thing performs 3 request per cycle - 18 per hour with default interval.

```

Comment:
JFTR: when starting both bridge and ondilo thing (like on swtartup) are danger close to the limit 4/5 per second. If you have multiple things, you would exceed the rate limit?

Reply:
Using different initial delays reduces the risk.

2 Things would still cause 6 calls per second. I'm not sure, how strict the quota is and how much energy needs to be invested into this.

Reply:
Things has now an initial delay of 2 s while Bridge has 1 s.

There would be a bigger API request handler logic required, if users have several pools.
E.g. Poll Bridge in S0, Thing 1 in S1, Thing 2 in S2 ...
This could be implemented during the bigger Pool handler <=> Bridge Handler <=> ApiClient change that you are asking for.